frontend/src/components/Navbar/navbar.module.scss

```diff
@@ -53,7 +53,8 @@
 }
 
 .navbar-logo-dark img:hover {
-  filter: hue-rotate(-15deg) contrast(180%) brightness(70%);
+  // filter: hue-rotate(800deg) contrast(180%) brightness(70%);
```

Comment:
let's remove the commented line

Reply:
Done with the requested changes @Kajol-Kumari , please review :)